app/models/vacancy.rb

```diff
@@ -268,6 +268,16 @@ def delete_documents
     self.documents.each { |document| DocumentDelete.new(document).delete }
   end
 
+  def new_information
```

Comment:
Maybe there is a cleaner way of doing this. We want to add a link for every section that has new information.

Reply:
I would use a rails helper for this

Reply:
What kind of rails helper would help with this?

Reply:
Have a look at the helper methods defined in `app/helpers/vacancies_helper.rb`